app/states/tx-history-state.js

```diff
@@ -1,27 +1,32 @@
-import { observable, action, runInAction } from 'mobx'
+// @flow
+import { observable, action, runInAction, type IObservableArray } from 'mobx'
 
-import { getTxHistory } from '../services/api-service'
+import { getTxHistory, type TransactionDelta } from '../services/api-service'
 
-// uncomment when zen node have pagination support for comments
-// const BATCH_SIZE = 50
+export type ObservableTransactionResponse = {
```

Comment:
cc @goldylucks needing to do this (i.e. create a new type that wraps  the deltas array in IObservableArray) to make flow-runtime happy. The need to do this might become annoying since mobx seems to decorate arrays. Not wrapping it like this throws a runtime error and causes a white screen.